android/build.gradle

```diff
@@ -58,13 +58,13 @@ allprojects {
         // TODO: Get rid of this, when no deps use it.
         jcenter() {
             content {
-                // Allow jcenter for only me.relex:photodraweeview:1.0.0:
+                // Allow jcenter for only me.relex:photodraweeview:1.1.3:
```

Comment:
Regarding the `jcenter` exception for `photodraweeview` (see
    android/build.gradle in this commit), I tried removing the exception
    entirely, hoping that 1.1.3 might be available in the non-jcenter
    repos consulted by the build, even though 1.0.0 was not. But no, I
    again got an error that `photodraweeview` could not be found, so it
    looks like we still need to make an exception.

Huh interesting. That seems at variance with the latest commit in react-native-photo-view: https://github.com/alwx/react-native-photo-view/pull/208

Perhaps the build worked for that person because he actually still had the `jcenter` dependency in the app's own build.gradle.

Reply:
Yeah, odd.

> Perhaps the build worked for that person because he actually still had the `jcenter` dependency in the app's own build.gradle.

That sounds plausible. I read https://mvnrepository.com/artifact/me.relex/photodraweeview as saying that `photodraweeview` is published to the "Central" repository (as in `mavenCentral`) only at version 2.1.0:

<img width="1100" alt="image" src="https://user-images.githubusercontent.com/22248748/196639972-afe26202-b04e-4e46-b6df-39357e88660b.png">

And version 1.1.3 (the version apparently used by react-native-photo-view) is published in a "Spring Plugins" repo:

<img width="1096" alt="image" src="https://user-images.githubusercontent.com/22248748/196640172-232edb51-059c-4a45-a0d6-6237e700e3ba.png">

And that (strangely) the only version published in JCenter is 2.0.0:

<img width="1100" alt="image" src="https://user-images.githubusercontent.com/22248748/196640372-05d4f291-0f2b-4df6-97ba-b022917f0928.png">

Reply:
> And that (strangely) the only version published in JCenter is 2.0.0

Strange because the my investigation that you quoted suggested we were getting 1.1.3 from JCenter…

And I think maybe that's true? https://jcenter.bintray.com/me/relex/photodraweeview/1.1.3/photodraweeview-1.1.3.aar serves a file. 🤷

Reply:
> I read https://mvnrepository.com/artifact/me.relex/photodraweeview as saying that `photodraweeview` is published to the "Central" repository (as in `mavenCentral`) only at version 2.1.0:

Neat, seems like a useful resource!

I guess one has to hope that its incompleteness as to what's in JCenter is related to limitations in the latter, perhaps to its partial shutdown, and doesn't affect other repos we'll want to use.